policies/release-requirements.md

```diff
@@ -0,0 +1,92 @@
+Release Requirements Policy
+===========================
+
+The OpenSSL project team creates the following three types of OpenSSL software
+releases:
+
+- [stable] releases
+- [alpha] (pre-)releases
+- [beta] (pre-)releases
+
+This policy defines the requirements on the state of a branch in the source
+tree that must be met before a release from that branch can be done.
+
+Alpha Pre-releases
+------------------
+
+As this is just a preview release for testing things that have been worked
+on in the development branch, the requirements are minimal.
+
+- The CI must pass on the tip of the development branch before the release
+  commits were added to the tree.
+
+Beta Pre-releases
+-----------------
+
+The API and ABI should be stable and the source code should be feature complete
+by the first beta pre-release. The following release requirements apply to beta
+pre-releases.
+
+- The code is functionaly complete in regards to the particular release
+  objectives as set by OMC and OTC.
+- There is no remaining refactoring required by OMC or OTC for the release.
+- There are no remaining API changes required for the release.
```

Comment:
I assume that we can, and do, change the API after the beta, but only in exceptional cases.

Reply:
If we have planned or expected or anticipated API changes then we have not reached **beta** state. Remember we are operating under the more common definition of beta rather than the previous practice which was effectively later-rolling-alpha releases. 

We can always handle things by an explicit exception vote - but we should not be planning or writing a policy that we expect to never actually follow.

Reply:
As far as I know, we've never been able to actually do this.

Reply:
Anything that is outside of this policy requires exception vote. I think explicitly mention that escape route isn't necessary as that applies to every policy we have.

Reply:
I'd like to see smth like "there are no known/expected/planned remaining API changes"

Reply:
Everything is about thinks we know at the point of time we are doing the release. We cannot see into the future.